src/System.Private.CoreLib/shared/System/Runtime/Serialization/IFormatterConverter.cs

```diff
@@ -23,6 +24,6 @@ public interface IFormatterConverter
         double ToDouble(object value);
         decimal ToDecimal(object value);
         DateTime ToDateTime(object value);
-        string ToString(object value);
+        string? ToString(object value);
```

Comment:
Interesting, did you decide this based on the same conclusions as `Object.ToString()`

Reply:
Yes, FormatterConverter calls ToString on the object.